### DISCUSSION TITLE
Add option to select Audio Format/Sample Rate

### DISCUSSION THREAD

**Discussion by BatJoe** [LizardByte/community#276] [Sunshine]
When using Virtual Speakers, like the Steam Streaming Speakers. Adding an option in Sunshine under Audio/Video where you can set the Default Audio Format/Sample Rate would be helpful.

Currently with no options it sets to 16 bit @ 48k, which can introduce audio crackle/stutter in certain games. Having other options such as 16bit @ 44.1k can alleviate many issues.

Having an option to override and set your own format/sample rate in Sunshine would help.

**Comment by xCISACx** [comment]
This is 100% needed.
I get terrible audio crackling all across the board that makes Sunshine unusable when Steam Streaming Speakers is set to 16-bit 48K.
I can't even change the format because the sound is gone if I do!
Upon restarting Sunshine, it's back to the default format which crackles terribly.
Please add this option...

**Comment by snicmtl** [comment]
So happy to find the possible reason to my audio crackling. Any reason this would of shown up on newer releases and not previous ones?

**Comment by massej** [comment]
That would be a good option, I have the same issue with 24 bits everytime i start a game i need to drop down to 16 bits....
